books/free-programming-books-langs.md

```diff
@@ -599,12 +599,17 @@ Books on general-purpose programming that don't focus on a specific language are
 
 ### COBOL
 
+* [COBOL: A Self Instructional Manual](https://babel.hathitrust.org/cgi/pt?id=mdp.39015000482466&view=1up&seq=14&skin=2021) - James A. Saxon 
+* [Cobol: From Micro To Mainframe](https://vdoc.pub/download/cobol-from-micro-to-mainframe-3rd-edition-5t4osmo5p540) - Robert T. Grauer, Carol Vazquez Villar, Arthur R. Buss (PDF)
```

Comment:
This appears not to be an authorized distribution. Please see docs/CONTRIBUTING.md guidelines.